contracts/contracts/oracle/OETHOracleUpdater.sol

```diff
@@ -0,0 +1,127 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import { IOracleReceiver } from "./IOracleReceiver.sol";
+import { IVault } from "../interfaces/IVault.sol";
+import { AggregatorV3Interface } from "../interfaces/chainlink/AggregatorV3Interface.sol";
```

Comment:
I don't think AggregatorV3Interface is used?

Reply:
AggregatorV3Interface  is not used. I'll remove it.

Apparently Slither can detect unused imports. I'll see why it's not detecting ours.

Reply:
Yeah, I was wondering how this got through the automated tools. I should have chased this down more.